usr.sbin/daemon/daemon.c

```diff
@@ -612,26 +612,26 @@ listen_child(int fd, struct log_params *logpar)
 		}
 		/* Wait until the buffer is full. */
 		if (bytes_read < LBUF_SIZE - 1) {
-			return 1;
+			return true;
                 }
 		do_output(buf, bytes_read, logpar);
 		bytes_read = 0;
-		return 1;
+		return true;
 	} else if (rv == -1) {
 		/* EINTR should trigger another read. */
 		if (errno == EINTR) {
-			return 1;
+			return true;
 		} else {
 			warn("read");
-			return 0;
+			return true;
 		}
 	}
 	/* Upon EOF, we have to flush what's left of the buffer. */
 	if (bytes_read > 0) {
 		do_output(buf, bytes_read, logpar);
 		bytes_read = 0;
 	}
-	return 0;
+	return false;
```

Comment:
(this is for o returns, not just this one)

doesn't our style guide say to use

```c
return (value);
```

?